Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1568,15 +1568,21 @@ void MediaPlayerPrivateGStreamerBase::emitPlayReadySession(PlayreadySession* ses
 #endif // USE(PLAYREADY)
 
 #if USE(OPENCDM)
-void MediaPlayerPrivateGStreamerBase::emitOpenCDMSession()
+CDMSessionOpenCDM* MediaPlayerPrivateGStreamerBase::getOpenCDMSession()
 {
-    if (!m_cdmSession)
-        return;
 #if ENABLE(LEGACY_ENCRYPTED_MEDIA)
-    CDMSessionOpenCDM* cdmSession = static_cast<CDMSessionOpenCDM*>(m_cdmSession);
+    return static_cast<CDMSessionOpenCDM*>(m_cdmSession);
```

Comment:
dynamic_cast is not allowed: `error: ‘dynamic_cast’ not permitted with -fno-rtti`